osu.Game/Overlays/AccountCreation/ScreenEntry.cs

```diff
@@ -132,10 +133,10 @@ private void load()
 
             textboxes = new[] { usernameTextBox, emailTextBox, passwordTextBox };
 
-            usernameDescription.AddText("This will be your public presence. No profanity, no impersonation. Avoid exposing your own personal details, too!");
+            usernameDescription.AddText(AccountCreationStrings.UsernameDescription);
 
-            emailAddressDescription.AddText("Will be used for notifications, account verification and in the case you forget your password. No spam, ever.");
-            emailAddressDescription.AddText(" Make sure to get it right!", cp => cp.Font = cp.Font.With(Typeface.Torus, weight: FontWeight.Bold));
+            emailAddressDescription.AddText(AccountCreationStrings.EmailDescription1);
+            emailAddressDescription.AddText(AccountCreationStrings.EmailDescription2, cp => cp.Font = cp.Font.With(Typeface.Torus, weight: FontWeight.Bold));
```

Comment:
This I'm still unsure about. For now it's *probably* okay given we don't support any right-to-left scripts. But eventually this is gonna break. If you have to break a coherent string in half due to formatting then that's a bit of a giveaway that it may not localise correctly.

Reply:
Probably fine for now.